backend/estimate-service/pom.xml

```diff
@@ -64,6 +65,16 @@
             <artifactId>tracer</artifactId>
             <version>2.0.0-SNAPSHOT</version>
         </dependency>
+        <dependency>
+            <groupId>org.egov.services</groupId>
+            <artifactId>services-common</artifactId>
+            <version>1.0.1-SNAPSHOT</version>
+        </dependency>
+        <dependency>
```

Comment:
Don't add this. Only services-common should be enough

Reply:
done